doctr/transforms/functional/base.py

```diff
@@ -42,3 +45,102 @@ def crop_boxes(
     is_valid = np.logical_and(boxes[:, 1] < boxes[:, 3], boxes[:, 0] < boxes[:, 2])
 
     return boxes[is_valid]
+
+
+def expand_line(line: np.ndarray, target_shape: Tuple[int, int]) -> Tuple[int, int]:
```

Comment:
can you add minimal comments here to describe what the function does please ?

Reply:
good idea!